lua/supermaven-nvim/util.lua

```diff
@@ -186,4 +187,49 @@ function M.to_next_word(str)
   return str
 end
 
+-- Flattenning table for all versions
```

Comment:
nit: spelling - "Flattenning" should be "Flattening"

Reply:
Whoops, missed it.

It's fixed now.